### ISSUE TITLE
Relies on Symbol existing

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [educastellano/bidirectional-map#1]
Breaks in IE 11 since that browser doesn't have Symbol.

If you don't want to support browsers without Symbol, could you put a notice in the README.md please?

**Comment by JoshuaKGoldberg** [comment]
Never mind, wrong repository 💖

**Comment by JoshuaKGoldberg** [comment]
Huh, your build step adds it in.

Constructor in `index.js`:

```javascript
    constructor(object=null) {
        this._map = new Map()
        this._reverse = new Map()
        if (object) {
            for (let attr of Object.keys(object)) {
                this.set(attr, object[attr])
            }
        }
    }
```

Constructor in `dist/index.js`:

```javascript
    function BidirectionalMap() {
        var object = arguments.length <= 0 || arguments[0] === undefined ? null : arguments[0];

        _classCallCheck(this, BidirectionalMap);

        this._map = new Map();
        this._reverse = new Map();
        if (object) {
            var _iteratorNormalCompletion = true;
            var _didIteratorError = false;
            var _iteratorError = undefined;

            try {
                for (var _iterator = Object.keys(object)[Symbol.iterator](), _step; !(_iteratorNormalCompletion = (_step = _iterator.next()).done); _iteratorNormalCompletion = true) {
                    var attr = _step.value;

                    this.set(attr, object[attr]);
                }
            } catch (err) {
                _didIteratorError = true;
                _iteratorError = err;
            } finally {
                try {
                    if (!_iteratorNormalCompletion && _iterator["return"]) {
                        _iterator["return"]();
                    }
                } finally {
                    if (_didIteratorError) {
                        throw _iteratorError;
                    }
                }
            }
        }
    }
```

**Comment by JoshuaKGoldberg** [comment]
Also, if `var _iterator = Object.keys(object)[Symbol.iterator]()` is undefined (as it is in IE), this will throw.

@educastellano is there anything stopping the package from using a traditional:

```javascript
for (let i in object) {
    if ({}.hasOwnProperty.call(object, i)) {
        this.set(i, object[i]);
    }
}
```

**Comment by educastellano** [comment]
Cool, thanks 👍 

Released in @1.0.4
